sdk/python/tests/integration/e2e/test_universal_e2e.py

```diff
@@ -0,0 +1,134 @@
+import math
+from datetime import datetime, timedelta
+from typing import Optional
+
+import pandas as pd
+import pytest
+from pytz import utc
+
+from feast import FeatureStore, FeatureView
+from tests.integration.feature_repos.universal.data_sources.bigquery import (
+    BigQueryDataSourceCreator,
+)
+from tests.integration.feature_repos.test_repo_configuration import construct_feature_store
+
+parameters = [
+    ("bigquery", BigQueryDataSourceCreator, "datastore"),
+]
+
+
+@pytest.mark.parametrize("params", parameters)
+@pytest.mark.skip(reason="Still working on this test")
+def test_e2e_consistency(params):
+    with construct_feature_store(params) as fs:
```

Comment:
I'm a bit lost here. It seems like

```
parameters = [
    ("bigquery", BigQueryDataSourceCreator, "datastore"),
]
```
is passed into 
```
construct_feature_store(params)
```
but `construct_feature_store` takes a `test_repo_config` as an input. 

Am I missing something :thinking:

Reply:
I updated this API to use the `TestRepoConfig` instead